### ISSUE TITLE
brew install fails with python 3.7

### ISSUE THREAD

**Issue by rokroskar** [SwissDataScienceCenter/renku-python#273]
Following instructions, `brew install` fails with python 3.7. The error log is scant but includes this:

```
Already downloaded: /Users/rok/Library/Caches/Homebrew/renku--ruamel.yaml--0.14.12.tar.gz
==> /usr/local/Cellar/renku/0.1.0.dev20180621/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/renku--ruamel.yaml-20180820-94148-1t3x1ph/ruamel.yaml-0.14.12
Last 15 lines from /Users/rok/Library/Logs/Homebrew/renku/31.pip:
  File "/usr/local/Cellar/renku/0.1.0.dev20180621/libexec/lib/python3.7/site-packages/pip/_internal/resolve.py", line 102, in resolve
    self._resolve_one(requirement_set, req)
  File "/usr/local/Cellar/renku/0.1.0.dev20180621/libexec/lib/python3.7/site-packages/pip/_internal/resolve.py", line 256, in _resolve_one
    abstract_dist = self._get_abstract_dist_for(req_to_install)
  File "/usr/local/Cellar/renku/0.1.0.dev20180621/libexec/lib/python3.7/site-packages/pip/_internal/resolve.py", line 209, in _get_abstract_dist_for
    self.require_hashes
  File "/usr/local/Cellar/renku/0.1.0.dev20180621/libexec/lib/python3.7/site-packages/pip/_internal/operations/prepare.py", line 298, in prepare_linked_requirement
    abstract_dist.prep_for_dist(finder, self.build_isolation)
  File "/usr/local/Cellar/renku/0.1.0.dev20180621/libexec/lib/python3.7/site-packages/pip/_internal/operations/prepare.py", line 126, in prep_for_dist
    self.req.run_egg_info()
  File "/usr/local/Cellar/renku/0.1.0.dev20180621/libexec/lib/python3.7/site-packages/pip/_internal/req/req_install.py", line 636, in run_egg_info
    command_desc='python setup.py egg_info')
  File "/usr/local/Cellar/renku/0.1.0.dev20180621/libexec/lib/python3.7/site-packages/pip/_internal/utils/misc.py", line 701, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip._internal.exceptions.InstallationError: Command "python setup.py egg_info" failed with error code 1 in /private/tmp/pip-req-build-cx6vzixd/

```

So it looks like another issue with `ruamel.yaml`. `pip install` from master works so maybe it's just a matter of updating the bottle?

**Comment by rokroskar** [comment]
resolved by 0.2.0rc1

**Comment by jirikuncar** [comment]
Fixed in 0.2.0rc1
